### PR TITLE
#487: use userIdForDisplay for the User Id column; display number of …

### PR DESCRIPTION
**Pull request by sudo-may**
…points requested

### CODE REVIEW

File: service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy [hunk 1]
```diff
@@ -705,6 +719,8 @@ class SkillApprovalSpecs extends DefaultIntSpec {
 
         def approvalsEndpointRes = skillsService.getApprovals(proj.projectId, 50, 1, 'requestedOn', false)
         List approvalItems = approvalsEndpointRes.data.sort({ it.userId })
+        println "rejected ${approvalItems[1]}"
+        println "rejected ${approvalItems[2]}"
```
Comment (rmmayo) [inline service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy:723]: did you mean to leave these `println` statements?

File: service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy [hunk 2]
```diff
@@ -726,14 +742,17 @@ class SkillApprovalSpecs extends DefaultIntSpec {
...
+            "TestProject1-skill1_${users[it]}"
+        }.sort()
+//        [
+//                "TestProject1-skill1_${users[0]}",
+//                "TestProject1-skill1_${users[1]}",
+//                "TestProject1-skill1_${users[2]}",
+//                "TestProject1-skill1_${users[3]}",
+//                "TestProject1-skill1_${users[4]}",
+//                "TestProject1-skill1_${users[5]}",
+//        ].sort()
```
Comment (rmmayo) [inline service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy:755]: should these commented out lines be deleted?

File: service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy (hunk 1 above)
Comment (sudo-may) [reply service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy:723]: nope, removed!

File: service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy (hunk 2 above)
Comment (sudo-may) [reply service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy:755]: yes, removed!

**Pull request merged**